ui/src/institutions/containers/DetailPageContainer.jsx

```diff
@@ -0,0 +1,139 @@
+import AddressList from '../../common/components/AddressList';
+import InstitutionPapers from './InstitutionPapers';
+import { newSearch } from '../../actions/search';
+import { INSTITUTION_PAPERS_NS } from '../../reducers/search';
+import RequireOneOf from '../../common/components/RequireOneOf';
+import GridLink from '../components/GridLink';
+import RorLink from '../components/RorLink';
+import InstitutionsNameVariantsList from '../components/InstitutionNameVariantsList';
+import InstitutionPeople from '../components/InstitutionPeople';
+import RelatedRecordsList from '../components/RelatedRecordsList';
+
+function DetailPage({ record }) {
+  const metadata = record.get('metadata');
+  const urls = metadata.get('urls');
+  const controlNumber = metadata.get('control_number');
+  const legacyIcn = metadata.get('legacy_ICN');
+  const addresses = metadata.get('addresses');
+  const hierarchies = metadata.get('institution_hierarchy');
+  const grid = metadata.get('grid');
+  const ror = metadata.get('ror');
+  const nameVariants = metadata.get('name_variants');
+  const parentInstitutions = metadata.get('parent_institutions');
+  const successorInstitutitons = metadata.get('successor_institutions');
+  const predecessorInstitutions = metadata.get('predecessor_institutions');
+  const subsidiaryInstitutions = metadata.get('subsidiary_institutions');
+
+  // add meta description
+  return (
+    <>
+      <DocumentHead title={legacyIcn} />
+      <Row justify="center">
+        <Col className="mv3" xs={24} md={22} lg={21} xxl={18}>
+          <ContentBox
+            className="sm-pb3"
+            leftActions={<>{urls && <WebsitesAction websites={urls} />}</>}
+          >
+            <Row>
+              <Col>
+                <h2>{legacyIcn}</h2>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <InstitutionHierarchyList hierarchies={hierarchies} />
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <AddressList addresses={addresses} />
+              </Col>
+            </Row>
+            <RequireOneOf
+              dependencies={[
+                grid,
+                ror,
+                nameVariants,
+                parentInstitutions,
+                successorInstitutitons,
+                predecessorInstitutions,
+              ]}
+            >
+              <Row className="mt3">
+                <Col>
+                  {grid && (
+                    <Row>
+                      <GridLink grid={grid} />
+                    </Row>
+                  )}
+                  {ror && (
+                    <Row>
+                      <RorLink ror={ror} />
+                    </Row>
+                  )}
+                  <InstitutionsNameVariantsList nameVariants={nameVariants} />
+                  <RelatedRecordsList
+                    relatedRecords={parentInstitutions}
+                    label="Parent Institution"
```

Comment:
do this  and others need `pluralizeUnlessSingle`? 

cc @StellaCh

Reply:
yes